packages/tasks/src/local-apps.ts

```diff
@@ -83,15 +83,19 @@ function isLlamaCppGgufModel(model: ModelData) {
 }
 
 const snippetLlamacpp = (model: ModelData, filepath?: string): LocalAppSnippet[] => {
-	const command = (binary: string) =>
-		[
+	const command = (binary: string) => {
+		let snippet = [
```

Comment:
can you keep it a const, by swithing the place of the `.join()` at the very end, i.e. pushing to the array inside the `if`?